migration/migration/src/lib.rs

```diff
@@ -108,8 +123,43 @@ pub mod pallet {
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
 
+		/// Set Tokens (GAME & PLAY) balances with the same amount as native currency has
+		#[pallet::weight(5_000_000)]
+		pub fn tokens_airdrop(origin: OriginFor<T>) -> DispatchResult {
+			ensure_root(origin.clone()).is_ok();
+
+			if <TokensVersion<T>>::get() == StorageVersion::V2Imported {
+				return Ok(())
+			}
+			let mut accounts: u32 = 0;
+
+			// for (acc_id, acc_info) in pallet_balances::Account::<T>::iter() {				// --> contains no accounts
+			// for (acc_id, acc_info) in <T as pallet_balances::Config>::AccountStore::iter() {	// --> no iterator
+			for (acc_id, _acc_info) in <frame_system::Account::<T>>::iter() {				
```

Comment:
shouldn't blessed accounts be excluded from this?

Reply:
Could you elaborate on this?